sql/moz-fx-data-shared-prod/stub_attribution_service_derived/dl_token_ga_attribution_lookup_v2/checks.sql

```diff
@@ -0,0 +1,5 @@
+#fail
+{{ is_unique(['dl_token', 'ga_client_id', 'stub_session_id', 'download_source']) }}
```

Comment:
If `ga_client_id` is at client level, would `first_seen_date` be unique as well?